README.md

```diff
@@ -80,32 +80,61 @@ metric exporters, service monitors, and custom dashboards.
+- creating the location for your local customization files
+- specifying customization variables and parameters in the customization files
+- using the `USER_DIR` environment variable to specify the location of the customization files
 
-The components that are deployed are highly customizable. The recommended
-approach for customization is to set a `USER_DIR` environment variable. `USER_DIR`
-points to a directory containing user-modified customization files which can exist
-outside of your repository. You can use customization files to check in
-environment-specific customizations into a separate Git repository, for example.
+### Create the Location for Customization Files
 
-`USER_DIR` must refer to a directory. The directory can include any or all of
-the following files in the structure below. Additional content is allowed, but
-will be ignored.
+The `USER_DIR` environment variable enables you to use a directory outside of the local repository to contain the customization files. By using a location outside of the repository, you can preserve your customizations when you deploy new versions of the monitoring and logging applications. You could also put the customization files in a location that is under version control.
+
+For example, to create a customization directory named *my_custom_files* that contains directories for monitoring and logging customization files:
+
+```bash
+mkdir -p ~/my_custom_files/monitoring
+mkdir -p ~/my_custom_files/logging
+```
+
+### Specifying Customization Variables and Parameters
+
+After you create the location for your customization files, you can customize the components that you deploy for monitoring and logging by specifying environment variables and Helm chart parameters in this set of customization files (in this example, contained in the *my_custom_files* root directory):
 
 ```text
-- user.env
-- [monitoring]
-  - user.env
-  - user-values-prom-operator.yaml
-  - user-values-pushgateway.yaml
-- [logging]
-  - user.env
-  - user-values-elasticsearch-open.yaml
-  - user-values-es-exporter.yaml
-  - user-values-fluent-bit-open.yaml
+/my_custom_files/user.env
+
+/my_custom_files/monitoring/user.env
+/my_custom_files/monitoring/user-values-prom-operator.yaml
+/my_custom_files/monitoring/user-values-pushgateway.yaml
+
+/my_custom_files/logging/user.env
+/my_custom_files/logging/user-values-elasticsearch-open.yaml
+/my_custom_files/logging/user-values-es-exporter.yaml
+/my_custom_files/logging/user-values-fluent-bit-open.yaml
+```
+
+You specify the environment variables in the `user.env` files and the Helm chart parameters in the `*.yaml` configuration files. Note that you can also specify environment variables on a command line, but specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments. 
+
+You do not have to specify values in all of these files. The deployment process ignores any files other than these.
```

Comment:
"ignores any files other than these" -> "ignores any other files in the USER_DIR other than these"

Reply:
Actually, there are some other files that are used and, over time, I think we can expect the list of "used" files to grow.  To avoid having to update this section every time and/or to allow "undocumented" support for some files, I suggest removing the statement that "any other files in the USER_DIR...will be ignored.".  Let's reword things to indicate that these are some examples of the files that might be in the USER_DIR.  In addition, let's move the "definitive" list of supported files (and their purpose) the the lower-level readme files.  In the README, I think it would be sufficient to highlight a couple of the files and their use, mostly to demonstrate the naming patterns/conventions.  I think we definitely want to describe the user.env files here (e.g. top-level user.env used by both, lower-level ones used by one side or the other).  I think we also want to discuss the samples here as well.  Basically, this discussion should explain the higher-level concepts and provide examples but not really document any particular use-case.  The corresponding lower-level README files _should_ document the supported files.  Similarly, the README files in the various samples should cover each specific use-case.

Reply:
My concern with listing and documenting the files in the lower-level READMEs is repetition of information and maintenance. If the files are listed in the sample READMEs and new ones are added, the information would have to be updated in multiple places, not just here. The samples are discussed here, but I'm adding a subheading to make the information easier to find.

Reply:
I guess I was suggesting that we only document the files in the lower-level README files and those files should all include something like "Read the customization section of the main README file for **_how_** to handle customizations...and come back here to read _**what**_ files you can use".  That would limit "duplication".  But I understand the need to avoid having to update multiple README files if some new sample comes along that is a little different than the others.  It might make sense to step back and review what files and settings are used across the samples to see where they line up and where the conflict.